### PR TITLE
docs(splitter): Docs for SizeChanged and CollapsedChanged events

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/splitter/events.md [hunk 1]
```diff
@@ -145,9 +145,94 @@ Try resizing any of the panes by dragging the splitbars
 }
 ````
 
+## SizeChanged
+
+`SizeChanged` is triggered when the `Size` parameter of the corresponding pane is changed.
```
Comment (svdimitr) [inline components/splitter/events.md:150]: The `eventName` is triggered

File: components/splitter/events.md [hunk 2]
```diff
@@ -145,9 +145,94 @@ Try resizing any of the panes by dragging the splitbars
...
+        PaneSize = size;
+        Console.WriteLine("Pane 1 size was changed. Current size: " + PaneSize);
+    }
+}
+````
+
+
+## CollapsedChanged
+
+`CollapsedChanged` is triggered when the `Collapsed` parameter of the corresponding pane is changed.
```
Comment (svdimitr) [inline components/splitter/events.md:194]: The same review as in the `SizeChanged` event

**Review by svdimitr — CHANGES_REQUESTED** [review]

**Comment by svdimitr** [comment]
Apart from the two small changes requested, I think that we should make a new structure for the Splitter component. Maybe we should go for something similar to the Grid, where we have a dedicated folder for the Columns, here we might have it for the Pane.

File: components/splitter/events.md (hunk 1 above)
Comment (ntacheva) [reply components/splitter/events.md]: Updated

File: components/splitter/events.md (hunk 2 above)
Comment (ntacheva) [reply components/splitter/events.md]: Updated

**Review by svdimitr — APPROVED** [review]

**Pull request merged**